Actions/DetermineProjectsToBuild/action.yaml

```diff
@@ -0,0 +1,35 @@
+name: Determine Projects to Build
+author: Microsoft Corporation
+inputs:
+  shell:
+    description: Shell in which you want to run the action (powershell or pwsh)
+    required: false
+    default: powershell
+  parentTelemetryScopeJson:
+    description: Specifies the parent telemetry scope for the telemetry signal
+    required: false
+    default: '7b7d'
+outputs:
+  ProjectsJson:
+    description: An array of AL-Go projects in compressed JSON format
+    value: ${{ steps.determineProjectsToBuild.outputs.ProjectsJson }}
+  ProjectDependenciesJson:
+    description: An object that holds the project dependencies in compressed JSON format
+    value: ${{ steps.determineProjectsToBuild.outputs.ProjectDependenciesJson }}
+  BuildOrderJson: 
+    description: An array of objects that determine that build order, including build dimensions
+    value: ${{ steps.determineProjectsToBuild.outputs.BuildOrderJson }}
+runs:
+  using: composite
+  steps:
+    - name: Determine Projects to Build
+      shell: ${{ inputs.shell }}
+      id: determineProjectsToBuild
+      run: try { 
+          $changedFiles = ${{ github.action_path }}/GetModifiedFiles.ps1 -token ${{ github.token }}
```

Comment:
Did you verify this works? I actually thought the `github.token` was inaccessible to GitHub Actions. From the [docs](https://docs.github.com/en/actions/learn-github-actions/contexts#github-context): 

_Note: This context property is set by the Actions runner, and is only available within the execution steps of a job. Otherwise, the value of this property will be null._

Might be okay since you're accessing it from the execution steps of a GitHub Action?

Reply:
I'm double-checking...

Reply:
Tested in BuildOrder project (see [here](https://github.com/mazhelez/BuildOrder/pull/2)).

Churn-based build in the PR Handler works as expected.